apps/call.go

```diff
@@ -58,6 +58,7 @@ type CallRequest struct {
 	Values map[string]interface{} `json:"values,omitempty"`
```

Comment:
Is the corresponding ticket: https://mattermost.atlassian.net/browse/MM-31118 ?

Reply:
yes.